maths/series/geometric_mean.py

```diff
@@ -0,0 +1,55 @@
+def is_geometric_series(series: list) -> bool:
+    """
+    checking whether the input series is geometric series or not
+
+    >>> is_geometric_series([2, 4, 8])
+    True
+    >>> is_geometric_series([3, 6, 12, 24])
+    True
+    >>> is_geometric_series([1, 2, 3])
+    False
+
+    """
+    if len(series) == 1:
+        return True
+    common_ratio = series[1] / series[0]
```

Comment:
Will fail with empty series.

Reply:
Also if series[0] is 0 will fail as well

Reply:
> Will fail with empty series.

for that I  have check the size of list. If it is zero then It will raise an error. 

```python
if len(series) == 0:
     raise ValueError("Input list must be a non empty list")
```
Would it be fine or  ... how can I do it in a better way ?

Reply:
Right, didn't notice.